core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -565,8 +572,9 @@ fi
 
 # check for missing tpcds_kit.zip file
 install_hadoop_regr_test_env --check
+INSTALL_TPCDS=0
 if [ $? -ne 0 ]; then
-  exit 1
+  INSTALL_TPCDS=1
```

Comment:
You are correct.  I thought I tested this version of the script but apparently not. Will fix it.